setup.py

```diff
@@ -84,7 +85,7 @@ def alltests():
 
 
 setup(name='zope.container',
-      version='4.2.0.dev0',
+      version=read('version.txt'),
```

Comment:
Again, I don't like the external version file vs. having other uses query via `pkg_resources`.